en/config/actuators.md

```diff
@@ -79,6 +79,34 @@ The `X`, `Y`, `Z` positions are in [FRD coordinate frame, relative to the _centr
 Note, this may not be the same as the position of the flight controller!
 :::
 
+#### Geometry: Helicopter
+
+The geometry for a [Generic Helicopter (with Tail ESC)](../airframes/airframe_reference.md#copter_helicopter_generic_helicopter_(tail_esc)) is shown below.
+
+This is the only helicopter configuration supported, and allows for a single main rotor with any type of swash-plate and an ESC tail rotor.
+
+![Geometry: helicopter](../../assets/config/actuators/qgc_geometry_helicopter.png)
+
+The motors have no configurable geometry:
+
+- `Rotor (Motor 1)`: The main rotor
+- `Yaw tail motor (Motor 2)`: The tail rotor
+
+Swash plate servos:
+- Set the number of servos
+
+  For each servo set:
+  
+  - `Angle`: Angle measured clockwise, with `0` pointing forwards.
+  - `Arm Length (relative)`: ?
+  - `Trim`: Offset individual servo positions (generally not needed).
```

Comment:
Again, I don't understand the geometry, so don't get how these are offset? Again, it could be like all servos are supposed to be at the same fixed radius at their specified angle, but if not, you can use this to compensate?